app/database/db.go

```diff
@@ -245,11 +245,13 @@ func (conn *DB) UpdateColumn(attrs ...interface{}) *DB {
 
 // SubQuery returns the query as sub query
 func (conn *DB) SubQuery() interface{} {
+	mustNotBeError(conn.Error())
```

Comment:
Why panicing (on purpose on a public interface 😕) instead of letting it flow to the more general db request? Ah maybe it will not as it is exported to a string?

Reply:
Here it is a fatal error. This should never happen in a correct program.